modules/bmo-troubleshooting-latency-when-deleting-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ipi-install-troubleshooothing-latency-when-deleting-a-baremetalhost-resource_{context}"]
+
+= Troubleshooting latency when deleting a BareMetalHost resource
+
+When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host with a process known as cleaning. If cleaning does not succeed, deleting the `BareMetalHost` resource takes considerable time and may not fully delete. The deleting process retries three times.
+
+[IMPORTANT]
+====
+Do not remove finalizers from `BareMetalHost` resources.
```

Comment:
You had a longer note in the QE approved PR, https://github.com/openshift/openshift-docs/pull/80775/files#diff-44ef2609fd9567f86debc306604b0dadb4b9e5b8dfbea87cd267342c754e8f58R20